.github/workflows/prov-compat-label.yml

```diff
@@ -113,10 +113,6 @@ jobs:
             name: openssl-3.0,
             dir: branch-3.0,
             tgz: branch-3.0.tar.gz,
-          }, {
-            name: openssl-3.1,
-            dir: branch-3.1,
-            tgz: branch-3.1.tar.gz,
```

Comment:
Let's not remove 3.1 as part of the urgent fix here. Please can we move that change into a regular PR?

Reply:
@mattcaswell Is it really necessary? The branch is not going to be changed anymore, there is no point in keeping it. I can also split the PR into the CI changes and the actual urgent fix.

Reply:
Right - that's what I meant split the PR to separate out the urgent bit. It was not immediately obvious to me whether the 3.1 change was the right thing to do in light of the 140-3 validation. It seems prudent to at least split that bit out into a regular PR so that it gets the necessary attention.